mlir/lib/Dialect/AMDGPU/IR/AMDGPUDialect.cpp

```diff
@@ -24,6 +25,7 @@
 #include "mlir/IR/PatternMatch.h"
 #include "mlir/IR/TypeUtilities.h"
 #include "llvm/ADT/TypeSwitch.h"
+#include "llvm/IR/Metadata.h"
```

Comment:
Stray include?

Reply:
I have no clues why VSCode would add it itself. Any ideas?

Reply:
@bjacob knows a flag to disable this

Reply:
`-header-insertion=never`

Reply:
@bjacob life-saver!